lib/grn_nfkc.h

```diff
@@ -35,6 +35,7 @@ typedef struct {
   grn_nfkc_compose_func compose_func;
   grn_bool include_removed_source_location;
   grn_bool report_source_offset;
+  grn_bool unify_alphabet_diacritical_mark;
```

Comment:
We want to use `bool` not `grn_bool` for newly written code. See also: #1638

```suggestion
  bool unify_alphabet_diacritical_mark;
```

Reply:
fix: [20dff36](https://github.com/groonga/groonga/pull/1831/commits/20dff364377347d458dd94b874914a1c0df2c3e4) Thanks!